zerver/openapi/zulip.yaml

```diff
@@ -17355,7 +17355,7 @@ components:
                   email:
                     type: string
                     description: |
-                      Email of the user.
+                      Zulip API email of the user.
```

Comment:
This is the `display_recipient` information for direct messages, which again will be the "Zulip API email".